docs/index.asciidoc

```diff
@@ -59,4 +61,34 @@ actual encoding of the text and Logstash will convert it for you.
 
 For nxlog users, you may to set this to "CP1252".
 
-
+[id="plugins-{type}s-{plugin}-ecs_compatibility"]
+===== `ecs_compatibility`
+
+  * Value type is <<string,string>>
+  * Supported values are:
+    ** `disabled`: does not use ECS-compatible field names
```

Comment:
One final note (sorry for the comment drip ): in this case enabling/disabling ECS doesn't have to do with ECS-compatible field names, but it's all about where data gets put (target vs no-target).

```suggestion
    ** `disabled`: JSON document data added at root level
```